src/components/ebay-combobox/index.marko

```diff
@@ -23,7 +23,12 @@ static var itemIgnoredAttributes = ["text"];
         respect the "off" key. Tested and works in all other browsers properly as well.
         -->
         <input
-            ...processHtmlAttributes(input, ignoredAttributes)
+            ...processHtmlAttributes(
+                Object.assign(data, {
+                    id: input.id || component.id + "-input"
```

Comment:
Since we are doing marko4 syntax, I would change this to:
`input.id || component.getElId("input")`
See https://github.com/eBay/ebayui-core/blob/5.0.0/src/components/ebay-textbox/index.marko#L17